src/extensions/replay/sessionrecording.ts

```diff
@@ -1099,5 +1105,12 @@ export class SessionRecording {
      * */
     public overrideLinkedFlag() {
         this._linkedFlagSeen = true
+        this._reportStarted('linked flag override')
```

Comment:
```suggestion
        this._reportStarted('linked_flag_override')
```

just so that it matches the others